### ISSUE TITLE
No webview electron tag in selector-type-no-unknown rule

### ISSUE THREAD

**Issue by krhubert** [stylelint/stylelint#2762]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Feature requests

> Which rule, if any, is this issue related to?

selector-type-no-unknown

> What CSS is needed to reproduce this issue?

```css
webview {}
```

> What stylelint configuration is needed to reproduce this issue?

any

> Which version of stylelint are you using?

8.0.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`stylelint "*css" --syntax scss`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

Unexpected unknown type selector "webview".

**Comment by jeddy3** [comment]
@krhubert Thanks for using the template.

As [per the docs](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-type-no-unknown/README.md):

> This rule considers tags defined in the HTML, SVG, and MathML specifications to be known.

Use the [`ignoreTypes: ["/regex/", "string"]` optional secondary option](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-type-no-unknown/README.md#ignoretypes-regex-string) to ignore further tags.
